extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -181,6 +198,33 @@ export class DraggableDrawer extends AMP.BaseElement {
       },
       true /** callToInitialize */
     );
+
+    if (isPageAttachmentUiV2ExperimentOn(this.win)) {
+      const spacerEl = dev().assertElement(
+        this.element.querySelector('.i-amphtml-story-draggable-drawer-spacer')
+      );
+
+      // Handle click on spacer element to close.
+      spacerEl.addEventListener('click', () => {
+        this.close_();
+      });
+
+      // Update spacerElHeight_ on resize for drag offset.
+      const observer = new this.win.ResizeObserver((e) => {
```

Comment:
Does this fail in an acceptable way when not supported?

Reply:
It looks like we have a polyfill for it:
https://github.com/ampproject/amphtml/blob/master/third_party/resize-observer-polyfill/ResizeObserver.js